apps/test/unit/templates/rubrics/RubricContainerTest.jsx

```diff
@@ -872,4 +878,210 @@ describe('RubricContainer', () => {
+    stubFetchAiEvaluations(mockAiEvaluations);
+    stubFetchTeacherEvaluations(noEvals);
+    stubFetchTourStatus({seen: null});
+
+    const {getByText} = render(
+      <Provider store={store}>
+        <RubricContainer
+          rubric={defaultRubric}
+          studentLevelInfo={defaultStudentInfo}
+          teacherHasEnabledAi={true}
+          currentLevelName={'test_level'}
+          reportingData={{}}
+          open
+        />
+      </Provider>
+    );
+
+    await wait();
+
+    // Had to access this button indirectly because RTL can't find it by text
+    const skipButton = getByText(
+      'Getting Started with Your AI Teaching Assistant'
+    ).closest('div').childNodes[1];
+
+    fireEvent.click(skipButton);
+
+    await wait();
+
+    expect(sendEventSpy).to.have.been.calledWith(EVENTS.TA_RUBRIC_TOUR_CLOSED, {
+      step: 0,
+    });
+
+    sendEventSpy.restore();
+  });
+
+  it('sends event when user completes the tour', async function () {
+    const sendEventSpy = sinon.spy(analyticsReporter, 'sendEvent');
+    stubFetchEvalStatusForUser(readyJson);
+    stubFetchEvalStatusForAll(readyJsonAll);
+    stubFetchAiEvaluations(mockAiEvaluations);
+    stubFetchTeacherEvaluations(noEvals);
+    stubFetchTourStatus({seen: null});
+
+    const {getByText} = render(
+      <Provider store={store}>
+        <RubricContainer
+          rubric={defaultRubric}
+          studentLevelInfo={defaultStudentInfo}
+          teacherHasEnabledAi={true}
+          currentLevelName={'test_level'}
+          reportingData={{}}
+          open
+        />
+      </Provider>
+    );
+
+    await wait();
+
+    const nextButton = getByText('Next Tip');
+
+    fireEvent.click(nextButton);
+    await wait();
+    fireEvent.click(nextButton);
+    await wait();
+    fireEvent.click(nextButton);
+    await wait();
+    fireEvent.click(nextButton);
+    await wait();
+    fireEvent.click(nextButton);
+    await wait();
+
+    const doneButton = getByText('Done');
+    fireEvent.click(doneButton);
+    await wait();
+
+    expect(sendEventSpy).to.have.been.calledWith(
+      EVENTS.TA_RUBRIC_TOUR_COMPLETE,
+      {}
+    );
+
+    sendEventSpy.restore();
+  });
+
+  it('sends event when tour is restarted from ? button', async function () {
+    const sendEventSpy = sinon.spy(analyticsReporter, 'sendEvent');
+    stubFetchEvalStatusForUser(readyJson);
+    stubFetchEvalStatusForAll(readyJsonAll);
+    stubFetchAiEvaluations(mockAiEvaluations);
+    stubFetchTeacherEvaluations(noEvals);
+    stubFetchTourStatus({seen: true});
+    stubUpdateTourStatus({seen: false});
+
+    const {getByTestId, queryByText} = render(
+      <Provider store={store}>
+        <RubricContainer
+          rubric={defaultRubric}
+          studentLevelInfo={defaultStudentInfo}
+          teacherHasEnabledAi={true}
+          currentLevelName={'test_level'}
+          reportingData={{}}
+          open
+        />
+      </Provider>
+    );
+
+    await wait();
```

Comment:
this one looks a bit tricky to get rid of, so I'm fine with leaving it in for now